terraso_backend/apps/story_map/views.py

```diff
@@ -194,13 +194,13 @@ def handle_config_media(new_config, current_config, request):
     return new_config
 
 
-def invalid_media_type(config):
+def is_valid_media_type(config):
     if "chapters" in config:
         for chapter in config["chapters"]:
             media = chapter.get("media")
             if media and not (media["type"].startswith(("image", "audio", "video", "embedded"))):
```

Comment:
question: do you want to use `all()` to  match the behaviour of `is_valid_shared_data_type()`?